packages/utilities/src/dom.ts

```diff
@@ -209,6 +202,7 @@ export function getRect(element: HTMLElement | Window | null): IRectangle | unde
 /**
  * Identify element as a portal by setting an attribute.
  * @param element Element to mark as a portal.
+ * @public
```

Comment:
this is the "fix"

Reply:
I'm confused how a jsdoc tag removes a runtime error? Are you sure this error wasn't due to a local build error (i.e. utilities package was out of date)? I've never seen this error while developing and testing portals.

Reply:
You're probably right that it was `rush` and my local setup. Closing this with the real focus fix in #6308